react/components/Alert/index.js

```diff
@@ -53,19 +54,40 @@ class Alert extends Component {
     }
 
     return (
-      <div className={`vtex-alert flex justify-between f5 near-black ${classes}`}>
-        <div className="flex items-center">
-          {showIcon && <div><Icon color={color} size={18} /></div>}
+      <div
+        className={`vtex-alert flex justify-between f5 near-black ${classes}`}
+      >
+        <div className="flex-ns flex-grow-1">
+          <div className="flex items-center flex-grow-1">
+            {showIcon && (
+              <div>
+                <Icon color={color} size={18} />
+              </div>
+            )}
 
-          <div className={`${showIcon ? 'ph5 flex' : 'pr5'}`}>
-            {this.props.children}
+            <div className={`${showIcon ? 'ph5 flex' : 'pr5'}`}>
+              {this.props.children}
+            </div>
           </div>
-        </div>
 
-        {onClose &&
-          <div className="vtex-alert__close-icon pointer flex items-center pv2" onClick={onClose}>
+          {actionLabel && (
```

Comment:
actionLabel && onActionClick

se nao vai tentar chamar undefined quando clicar

Reply:
mas e se quem for usar não quiser que tenha uma action?

Reply:
@klzns não tem problema isso, o button nao deve quebrar se onClick for undefined.

só se for uma questão de API, forçar o usuario a adicionar um handler de click se for ter um botao, mas dai sao outros 500